fetch_bringup/launch/include/teleop.launch.xml

```diff
@@ -1,7 +1,10 @@
 <launch>
 
+  <arg name="joy_device" default="/dev/ps3joy"/>
+
   <node name="joy" pkg="joy" type="joy_node">
-  	<param name="autorepeat_rate" value="1"/>
+    <param name="autorepeat_rate" value="1"/>
+    <param name="joy/dev" value="$(arg joy_device)"/>
```

Comment:
I think this should be 
```
    <param name="dev" value="$(arg joy_device)"/>
```
otherwise the rosparam shows up like this: /joy/joy/dev